.changeset/spicy-teachers-dance.md

```diff
@@ -0,0 +1,13 @@
+---
+"@justeattakeaway/pie-eslint-config": patch
+---
+
+[Added] - Setup `pie-eslint-config` package
+[Added] - `pie-eslint-config` base and strict rules
+[Added] - `pie-eslint-config` tests
+[Changed] - Set the PIE repo to use the new `pie-eslint-config` strict rules
+[Removed] - Removed `eslint-config-fozzie` dependency
+[Changed] - Updated `pie-docs`, `pie-icons`, and `pie-icons-vue` ESLint config
```

Comment:
Would is be possible to generate new changesets for package-specific changes so those packages get versioned correctly?

Happy to give if a hand if you need it!

Reply:
I believe it is, however, I was wondering why, since we have this in the `README`:

> Changesets are only required for Major, Minor, and Patch changes that have an effect on consumers. Changes that don't affect consumers do not need to be versioned. Examples include linting, testing or CI updates.

Reply:
Generally I think most look okay, but some things shouldn't be added to this changeset file, as they don't relate directly to the `@justeattakeaway/pie-eslint-config` directly.

```
[Changed] - Lint `pie-docs`, `pie-icons`, `pie-icons-vue`, and `.eslintrc.js` source code
[Changed] - Add missing dependency on `pie-icons-vue`
[Changed] - Updated `pie-docs`, `pie-icons`, and `pie-icons-vue` ESLint config
```

If you still want to capture these changes, I'd recommend creating changesets for the affected packages. But as you mentioned, as they're internal changes, don't feel obliged. I personally just don't think they should be added to this changeset, as they'll get added to the release notes when this package is published 👍🏼